docs/content/preview/reference/configuration/yb-tserver.md

```diff
@@ -1952,6 +1832,115 @@ Starting from version 2.18, the default is `-1`. Previously it was `4`.
 
 {{< /note >}}
 
+
+##### --full_compaction_pool_max_threads
+
+{{% tags/wrap %}}
+
+
+Default: `1`
+{{% /tags/wrap %}}
+
+The maximum number of threads allowed for non-admin full compactions. This includes post-split compactions (compactions that remove irrelevant data from new tablets after splits) and scheduled full compactions.
+
+##### --full_compaction_pool_max_queue_size
+
+{{% tags/wrap %}}
+
+
+Default: `500`
+{{% /tags/wrap %}}
+
+The maximum number of full compaction tasks that can be queued simultaneously. This includes post-split compactions (compactions that remove irrelevant data from new tablets after splits) and scheduled full compactions.
+
+##### --auto_compact_check_interval_sec
+
+{{% tags/wrap %}}
+
+
+Default: `60`
+{{% /tags/wrap %}}
+
+The interval at which the full compaction task will check for tablets eligible for compaction (both for the statistics-based full compaction and scheduled full compaction features). `0` indicates that the statistics-based full compactions feature is disabled.
+
+##### --auto_compact_stat_window_seconds
+
+{{% tags/wrap %}}
+
+
+Default: `300`
+{{% /tags/wrap %}}
+
+Window of time in seconds over which DocDB read statistics are analyzed for the purpose of triggering full compactions to improve read performance. Both `auto_compact_percent_obsolete` and `auto_compact_min_obsolete_keys_found` are evaluated over this period of time.
+
+`auto_compact_stat_window_seconds` must be evaluated as a multiple of `auto_compact_check_interval_sec`, and will be rounded up to meet this constraint. For example, if `auto_compact_stat_window_seconds` is set to `100` and `auto_compact_check_interval_sec` is set to `60`, it will be rounded up to `120` at runtime.
```

Comment:
```suggestion
`auto_compact_stat_window_seconds` must be evaluated as a multiple of [auto_compact_check_interval_sec](#auto-compact-check-interval-sec), and will be rounded up to meet this constraint. For example, if `auto_compact_stat_window_seconds` is set to `100` and `auto_compact_check_interval_sec` is set to `60`, it will be rounded up to `120` at runtime.
```